### PR TITLE
Fix: flicker on first browser load in dark mode

### PR DESCRIPTION
**Pull request by yomybaby**
The target of this PR is #2193 

This PR:
- Fixed the initial browser loading flicker in dark mode.
  - Without this PR, there is flickering with a light background screen when the browser refreshes in dark mode.
- Changed the background color of `Sider` in dark mode.
- [fix & refactor theme settings to use globalThis.isDarkMode](https://github.com/lablup/backend.ai-webui/pull/2227/commits/aa254f741104bb9db730582c4bba1444fa96aa5c)
-  [patch mwc-select and mwc-textfield](https://github.com/lablup/backend.ai-webui/pull/2227/commits/491b8649feb003a0e230a1a28ea4f35dec9f19cd)

| Before | After |
|--------|--------|
| image  | image  | 

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: resources/theme.json [hunk 1]
```diff
@@ -33,6 +34,10 @@
       },
       "Table": {
         "headerBorderRadius": 0
+      },
+      "Layout": {
+        "lightSiderBg": "#F4F7FD",
+        "siderBg": "#2A2C30"
       }
```
Comment (yomybaby) [inline resources/theme.json:41]: We need to set the dark `siderBg` and `lightSiderBg` to support the feature of forcing the theme mode of `Sider`.
Also you can change and suggest color value for dark `siderBg`. @agatha197

File: resources/webui.css [hunk 2]
```diff
@@ -36,6 +65,23 @@ body {
...
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+body.dark-theme .splash-header .logo{
+  background-image: url("/manifest/backend.ai-text-bgdark.svg");
+}
+
+body.dark-theme .splash-header {
+  background-color: #141414;
```
Comment (agatha197) [inline resources/webui.css]:
```suggestion
  background-color: var(--general-colorBgContainer, #141414);
```

File: resources/webui.css [hunk 3]
```diff
@@ -24,6 +47,12 @@ body {
   box-shadow: 0px 0px 3px 3px rgba(0, 0, 0, 0.2);
 }
 
+body.dark-theme .splash {
+  background-color: #191919;
+  border: 1px solid #444;
```
Comment (agatha197) [inline resources/webui.css]: Let's use token variable first :)

```suggestion
  border: 1px solid var(--general-colorBorder, #444);
```

**Review by agatha197 — CHANGES_REQUESTED** [review]

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**